crates/rolldown_plugin_module_federation/src/lib.rs

```diff
@@ -263,6 +263,11 @@ impl Plugin for ModuleFederationPlugin {
     Ok(())
   }
 
+  // Make sure the plugin resolve_id hook called at first, avoid vite resolver resolved the shared module.
+  fn resolve_id_meta(&self) -> Option<rolldown_plugin::PluginHookMeta> {
+    Some(rolldown_plugin::PluginHookMeta { order: Some(rolldown_plugin::PluginOrder::Pre) })
```

Comment:
Does rollup have this concept `PluginOrder: 'pre'`?

Reply:
see https://rollupjs.org/plugin-development/#build-hooks